libs/wingc/src/type_check.rs

```diff
@@ -4520,6 +4638,13 @@ impl<'a> TypeChecker<'a> {
 
 				let mut property_variable = self.resolve_variable_from_instance_type(instance_type, property, env, object);
 
+				// Validate access modifier
+				// if !property_variable.type_.is_unresolved() {
+				// 	// TODO: this isn't good enough becaues `instance_type` might be an array so we need to "actual" class_type (Array). This is done for us in
```

Comment:
do we have a tracking issue?

Reply:
oops, forgot to delete this comment. This is a leftover comment from an earlier iteration. Resolved.